src/dotnet/APIView/APIViewWeb/Managers/SamplesRevisionsManager.cs

```diff
@@ -87,6 +113,18 @@ public async Task<SamplesRevisionModel> UpsertSamplesRevisionsAsync(ClaimsPrinci
             return await UpsertSamplesRevisionsAsync(user, reviewId, sample, revisionTitle, FileName);
         }
 
+        public async Task UpdateSamplesRevisionAsync(ClaimsPrincipal user, string reviewId, string sampleRevisionId, string newContent = null, string newTitle = null)
+        {
+            var samplesRevision = await _samplesRevisionsRepository.GetSamplesRevisionAsync(reviewId, sampleRevisionId);
```

Comment:
Shouldn't we check to ensure `samplesRevision` is not null? What if that sample was deleted by one user while someone else is trying to edit it? It's an edge case but still better to handle than failing with exception.